web/src/main/kotlin/no/nav/su/se/bakover/web/Application.kt

```diff
@@ -142,16 +141,19 @@ internal fun Application.susebakover(
 
     install(StatusPages) {
         exception<Tilgangssjekkfeil> {
-            when (it.pdlFeil) {
-                is PdlFeil.IkkeTilgangTilPerson -> {
+            when (it.feil) {
+                is KunneIkkeHentePerson.IkkeTilgangTilPerson -> {
                     call.audit("[Tilgangssjekk] Bruker har ikke tilgang til person ${it.fnr}")
-                    log.warn("[Tilgangssjekk] Bruker har ikke tilgang til person. Melding: ${it.message}", it.pdlFeil)
-                    call.respond(HttpStatusCode.Forbidden, ErrorJson(it.pdlFeil.message))
+                    log.warn("[Tilgangssjekk] Bruker har ikke tilgang til person. Melding: ${it.message}")
```

Comment:
Hva gir egentlig `it.message` her nå som du fjernet kallet til `RuntimeException`-konstruktøren?

Reply:
Ah, den var jeg egentlig på vei til å fjerne!